src/api/app/assets/javascripts/webui/notification.js

```diff
@@ -1,5 +1,5 @@
 function setSelectAllCheckbox() {
-  $('#select-all-notifications').change(function() {
+  $(document).on('change', '#select-all-notifications', function() {
```

Comment:
Why this has been changed?

Reply:
Adding the event listener to the `document` makes the event to be triggered on **any** target element, even if the element is created after the script is loaded (like after an ajax call that changes the content). By doing this the `change` event is always listening to `#select-all-notifications` tag, so there is no longer need of calling back the `setSelectAllChackbox()` after the `_update.js.erb` renders through the ajax call.

Reply:
Could we make this clear in the body of the commit... or in another commit? Personally, only the title of the commit "Load notification js on purpose" looks not enough to fully understand the changes.

Reply:
I added a commit message detail here: https://github.com/openSUSE/open-build-service/pull/17553/commits/c215ea6517d672f76652249e49a83d82b048f538